lib/node_modules/@stdlib/_tools/scaffold/math-strided-real-typed-unary/data/src/alias__c.txt

```diff
@@ -16,6 +16,8 @@
 * limitations under the License.
 */
 
+/* THIS IS A GENERATED FILE. DO NOT EDIT DIRECTLY. */
```

Comment:
Also missing suffix support at L42. I believe we have precedent for this in other scaffolds.

Reply:
I couldn't find this in other scaffolds, but should we follow something like this:

```
{{C_TYPE}} y[] = { {{#if (eq C_TYPE "float")}}0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f{{else}}0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0{{/if}} };
```

Reply:
No, that is a bit complicated for what we need. We actually have somewhat of a precedent in the current scaffold.

https://github.com/stdlib-js/stdlib/blob/7df58771f61965f3c819af3ea3b9376fedd86a86/lib/node_modules/%40stdlib/_tools/scaffold/math-strided-real-typed-unary/scripts/scaffold.sh#L189

and gets used, e.g.,

https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/_tools/scaffold/math-strided-real-typed-unary/data/benchmark/c/benchmark__length__c.txt#L126

In which case, the referenced code above becomes

```
{{C_TYPE}} y[] = { 0.0{{C_TYPE_SUFFIX}}, 0.0{{C_TYPE_SUFFIX}}, 0.0{{C_TYPE_SUFFIX}}, 0.0{{C_TYPE_SUFFIX}}, 0.0{{C_TYPE_SUFFIX}}, 0.0{{C_TYPE_SUFFIX}}, 0.0{{C_TYPE_SUFFIX}}, 0.0{{C_TYPE_SUFFIX}} };
```

Reply:
Made the changes here and in `example__c.txt` as well. Thanks!